docs/install-conda-full.md

```diff
@@ -3,4 +3,5 @@ conda create -n ag python=3.10
 conda activate ag
 conda install -c conda-forge mamba
 mamba install -c conda-forge autogluon
+mamba install -c conda-forge "ray-tune >=2.6.3,<2.7" "ray-default >=2.6.3,<2.7"  # install ray for faster training
```

Comment:
Nit: `ray-tune~=2.6.3` is a bit more concise and specifies the same version range